slides.md

```diff
@@ -697,9 +697,9 @@ You can reference the [`--import` official documentation](https://nodejs.org/api
 
 # A12 The problem
 
-- Run in the terminal `node --test --experimental-test-coverage`.
+- Run in the terminal `node --test --experimental-test-coverage ./test/index.test.js`.
```

Comment:
are you sue that this is needed? the test runner should be able to find the tests automatically

Reply:
This change is in relation with the next line which reads "Watch the coverage not being 100%.".
When we run just `node --test --experimental-test-coverage`, it picks `solution.test.js` file as well and shows coverage as 100%.
So, adding `./test/index.test.js` makes sure coverage reported is not 100%.

Reply:
ah ok, makes sense then 👌